src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -238,6 +250,8 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
         font-size: 18px;
         color: var(--secondary-text-color);
       }
+
+      ${iconColorCSS}
```

Comment:
you should return an array, with `iconColorCSS` and the `css` above

Reply:
Done. Found two other places where I did the same for consistency.